app/components/Dashboard/PriceHistoryPanel/PriceHistoryChart.jsx

```diff
@@ -55,7 +55,9 @@ export default class PriceHistoryChart extends React.Component<Props, State> {
   render = (): React$Node => {
     const { className, prices, timeKey, priceKey } = this.props
 
-    return (
+    return isEmpty(prices) ? (
```

Comment:
Seems like prices is guaranteed to be an array right? If so, its better to just check the length because `isEmpty` will return true if `prices` is undefined/null.

Reply:
Prices is not an array - currently the response from cryptocompare is malformed and prices is an object...

Reply:
I updated `index.js` so that if pricing data is not an array it will fallback to an empty array and I changed this check to look for length like you suggested